frontend/VolunteerOne/constants/loginCredentials.js

```diff
@@ -0,0 +1,14 @@
+export default {
+  // username and passwords
+  "admin" : {
+    password: "password"
+  },
+
+  "Jessica Jones" : {
+    password: "1234"
+  },
+
+  "Jjones" : {
+    password: "password"
+  },
+};
```

Comment:
is it case sensitive? and should we add more? i think this should be good for the demo tho

Reply:
We can.. the only concern is that its coded for Jessica Jones to be the logged in user.